src/components/header/header.jsx

```diff
@@ -4,6 +4,7 @@ import './header.css';
 import { useAuth } from '@/hooks/use-auth';
 import Tweeter from '@/icons/tweeter';
 import NavElement from '@/UI/nav/nav';
+import {Link} from "react-router-dom";
```

Comment:
use single style of quotes in project. you are can download prettier to do it automaticly